### DISCUSSION TITLE
关于弹幕右键菜单

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2141] [综合讨论 / General]
现在弹幕右键菜单的复制也给做进悬停弹出的菜单里了，感觉再过段时间说不定屏蔽也给做进去，这右键菜单就没了。
![image](https://user-images.githubusercontent.com/49396421/129335240-44836ffb-54e3-49e4-8923-be31792ae369.png)
![image](https://user-images.githubusercontent.com/49396421/129336746-098132f6-0457-4a63-91a0-e306cb16ec97.png)

感觉不太妙啊。

**Comment by the1812** [comment]
危

**Reply by the1812** [reply]
感觉得研究下怎么留住这个点赞层了

**Reply by cqjjjzr** [reply]
> 感觉得研究下怎么留住这个点赞层了

或考虑将三个功能移入右键菜单？

**Reply by Geeyun-JY3** [reply]
> 或考虑将三个功能移入右键菜单？

不见得弹幕右键菜单以后还能留下，个人持悲观态度。要是生造个弹幕右键菜单，维护起来麻不麻烦就很难说了。

**Comment by cqjjjzr** [comment]
请问这一问题在 2.0 中有改善的可能吗？

**Reply by the1812** [reply]
2.0 上 preview 后还有不少问题要处理，等将来稳定后再看

**Reply by Geeyun-JY3** [reply]
感觉这和 #608 都不会太快做
